src/app/app.module.js

```diff
@@ -6,15 +6,25 @@ import ngAria from 'angular-aria';
 import lastSearchesService from './services/lastSearches';
 import openWeatherMapsService from './services/openWeatherMaps';
 import startPageModule from './start-page/start-page.module';
+import mainPageModule from './main-page/main-page.module';
+import config from './app.division';
 import lastSearches from './values/lastSearches';
 import currentWeather from './values/currentWeather';
 import "material-icons/iconfont/material-icons.scss";
 import "angular-material/angular-material.scss";
 
+import logo from './components/logo-letters/logo-letters';
```

Comment:
Please, sort this list of imports and split them by groups (third party libraries, services, components, styles, configurations, etc) and put an empty line in between.